### DISCUSSION TITLE
Can I add multi exclusion path on New-FalconMlExclusion ?

### DISCUSSION THREAD

**Discussion by NatchanonBR** [CrowdStrike/psfalcon#335] [Q&A]
I try

```
$EX = (Import-Csv D:\EXPath.csv)."Path"
New-FalconMlExclusion -Value $EX -ExcludedFrom blocking, extraction -GroupId xxxxxxx
```
Get Error "**New-FalconMlExclusion: Cannot process argument transformation on parameter 'Value'. Cannot convert value to type System.Strin**g."

Is there any other way to add multi exclusion path on New-FalconMlExclusion ?

**Comment by bk-cs** [comment]
You can't add multiple paths to a single ML exclusion. You need to create an exclusion for each path, or if they're similar, a glob that covers multiple paths.
